app/views/pages/connect.html.erb

```diff
@@ -12,7 +12,7 @@
   <section class="connect">
   <div class="connectionTypes">
     <a href="/saved_query_doc" class="connectionCard">
-      <h4>Browser</h4>
+      <h4>Click here to run queries on the AACT data from your web browser</h4>
       <p>
         Click here for instructions about how to use your browser to analyze the AACT data directly.
```

Comment:
This should be "Click here to run queries on the AACT data from your web browser."